docs/workloads/azure/backend/java_cqrs/requirements_java_cqrs.md

```diff
@@ -0,0 +1,68 @@
+---
+id: requirements_java_cqrs
+title: Requirements
+sidebar_label: Requirements
+hide_title: false
+hide_table_of_contents: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+defaultValue="windows"
+values={[
+{label: 'Windows', value: 'windows'},
+{label: 'macOS', value: 'macos'}
+]}>
+<TabItem value="windows">
+    <details open>
+        <summary>Requirements for running the API locally</summary>
+        <div>
+            <h5>Mandatory</h5>
+            <ul>
+                <li><a href="https://adoptopenjdk.net/">Install Java 11</a> and set JAVA_HOME environment variable to the directory java is installed and make sure MAVEN_HOME environment variable is not set in</li>
+                <li><a href="https://aka.ms/cosmosdb-emulator">CosmosDB Emulator 2.4.5+</a></li>
+                <li>An IDE</li>
+            </ul>
+            <h5>Optional</h5>
+            <ul>
+                <li><a href="https://www.jetbrains.com/idea/download/#section=windows">Intellij</a></li>
+            </ul>
+        </div>
+    </details>
+    <br />
+</TabItem>
+
+<TabItem value="macos">
+    <details open>
+        <summary>Requirements for running the API locally</summary>
+        <div>
+            <h5>Mandatory</h5>
+            <ul>
+                <li>homebrew</li>
+                <li>azure-cli: brew install azure-cli</li>
+                <li><a href="https://adoptopenjdk.net/releases.html">Install Java 11:</a> $ wget https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9%2B11.1/OpenJDK11U-jdk_x64_mac_hotspot_11.0.9_11.tar.gz</li>
```

Comment:
It would be nice to highlight the commands somehow. Probably wrap them in \<code\>. It would be same as ``